src/include/server/mir/input/input_event_transformer.h

```diff
@@ -26,10 +27,14 @@
 
 namespace mir
 {
+namespace time
+{
+class Clock;
+}
 namespace input
 {
 class EventBuilder;
-class InputEventTransformer
+class InputEventTransformer : public input::Seat
```

Comment:
this code is in the `input` namespace, so...

```suggestion
class InputEventTransformer : public Seat
```